scripts/lib-franklin.js

```diff
@@ -534,7 +535,8 @@ export function decorateTemplateAndTheme() {
 export function decorateButtons(element) {
   element.querySelectorAll('a').forEach((a) => {
     a.title = a.title || a.textContent;
-    if (a.href !== a.textContent) {
+    // skip buttons for 'Learn More'
+    if (a.textContent.startsWith('Learn More') === false && a.href !== a.textContent) {
```

Comment:
This seemed like a reasonable approach to skipping buttons. All the 'Learn More' usages are not buttons

Reply:
@agangam would it make sense to override the button styling as section style eg.  ‘no-button-decoration’ or something that is intuitive ? coz there are other areas where we have other color schemes for links.

Reply:
Yes that is correct.

Something seems off on the layout as this section should be --section-width and currently is taking over the view-width

<img width="1665" alt="Screenshot 2023-04-19 at 2 39 30 PM" src="https://user-images.githubusercontent.com/10811320/233169570-bee89ef6-c1c4-419b-980d-de615ec1927d.png">
<img width="1368" alt="Screenshot 2023-04-19 at 2 39 44 PM" src="https://user-images.githubusercontent.com/10811320/233169575-666361aa-282a-4f12-b5eb-a4afca86da24.png">

Reply:
Changed width to 1000px as the heading was going to over to next line with --section-width.

After changes : 
![image](https://user-images.githubusercontent.com/109620050/233506490-17fd65bc-5438-4628-a179-0bcbbba52021.png)